### PR TITLE
mypy: `pglookout/common.py` [BF-1560]

### PR DESCRIPTION
**Pull request by Mulugruntz**

### CODE REVIEW

File: pglookout/common.py [hunk 1]
```diff
@@ -4,40 +4,44 @@
...
     match = ISO_EXT_RE.match(value)
     if not match:
         match = ISO_BASIC_RE.match(value)
     if not match:
         raise ValueError(f"Invalid ISO timestamp {value!r}")
-    parts = dict(
-        (key, int(match.group(key) or "0")) for key in ("year", "month", "day", "hour", "minute", "second", "microsecond")
-    )
-    return datetime.datetime(tzinfo=None, **parts)
+    parts: dict[str, int] = {
```
Comment (kmichel-aiven) [inline pglookout/common.py:36]: Is this hint really needed ? It looks like mypy should be able to infer it.

Comment (Mulugruntz) [reply pglookout/common.py]: You're right. In an earlier version (keeping the `dict()` builtin) it was necessary. But using the `{}` syntax was already solving the issue, and I forgot to check without.
I pushed a new version that should have improved a few things :-)

**Comment by Mulugruntz** [comment]
This PR and the other ones related to type hinting will be handled in https://github.com/aiven/pglookout/pull/106.

**Comment by Mulugruntz** [comment]
This PR and the other ones related to type hinting will be handled in https://github.com/aiven/pglookout/pull/106.

**Pull request closed without merging**